srcpkgs/popt/template

```diff
@@ -1,15 +1,16 @@
 # Template file for 'popt'
 pkgname=popt
-version=1.18
+version=1.19
 revision=1
 build_style=gnu-configure
 hostmakedepends="autoconf tar automake libtool gettext-devel"
 short_desc="Command line option parsing library"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
-homepage="https://rpm.org/"
-distfiles="http://ftp.rpm.org/popt/releases/popt-1.x/popt-${version}.tar.gz"
-checksum=5159bc03a20b28ce363aa96765f37df99ea4d8850b1ece17d1e6ad5c24fdc5d1
+homepage="https://github.com/rpm-software-management/popt"
+changelog="https://github.com/rpm-software-management/popt/releases"
+distfiles="https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-${version}.tar.gz"
```

Comment:
This is the URL published with the 1.19 GitHub release (https://github.com/rpm-software-management/popt/releases/tag/popt-1.19-release), and it uses HTTPS instead of HTTP, so I thought it's a good idea to move to it.